BankChain/app/src/main/java/nl/tudelft/ewi/ds/bankchain/cryptography/ED25519.java

```diff
@@ -30,36 +32,36 @@ public static EdDSAPrivateKey generatePrivateKey() {
         return generatePrivateKey(seed);
     }
 
-    public static EdDSAPrivateKey generatePrivateKey(byte[] seed) {
```

Comment:
Please leave this public, we will use this outside of this package in the future.